### PR TITLE
namespace and hash url fixed for popups

### PR DESCRIPTION
**Pull request by himanshuchawla009**

### CODE REVIEW

File: src/handlers/Popup/PopupHandler.js [hunk 1]
```diff
@@ -3,14 +3,17 @@ import { EventEmitter } from 'events'
...
 
 class PopupHandler extends EventEmitter {
   constructor({ url, target, features, preopenInstanceId }) {
     super()
     const localUrl = url instanceof URL ? url : new URL(url)
-    if (localUrl.hash) localUrl.hash += `&isCustomLogin=${config.isCustomLogin}`
-    else localUrl.hash = `#isCustomLogin=${config.isCustomLogin}`
+    const iframeOrigin = getIFrameOriginObject()
+
+    if (localUrl.hash) localUrl.hash += `&isCustomLogin=${config.isCustomLogin}&namespace=${iframeOrigin.hostname}`
```
Comment (chaitanyapotti) [inline src/handlers/Popup/PopupHandler.js:15]: Move this to a function and use search Params addition

File: src/router.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ router.beforeResolve((to, from, next) => {
     if (!to.name.includes('Topup') && to.name !== 'walletTransfer') {
       Object.keys(from.query).forEach((key) => key === 'instanceId' || delete from.query[key])
     }
-    return next({ name: to.name, query: from.query, hash: to.hash, params: to.params })
+    return next({ name: to.name, query: from.query, hash: from.hash, params: to.params })
```
Comment (chaitanyapotti) [inline src/router.js:183]: I don't remember but there was a reason we did to.hash

**Review by chaitanyapotti — CHANGES_REQUESTED** [review]
Added comments

File: src/router.js (hunk 2 above)
Comment (himanshuchawla009) [reply src/router.js]: reverted

File: src/handlers/Popup/PopupHandler.js (hunk 1 above)
Comment (himanshuchawla009) [reply src/handlers/Popup/PopupHandler.js]: done

**Review by chaitanyapotti — APPROVED** [review]
Lgtm

**Pull request merged**